### PR TITLE
Update files with PEP8 standards in folder dojo/tools #003

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/gitleaks
dojo/tools/gosec
dojo/tools/govulncheck
dojo/tools/h1
dojo/tools/harbor_vulnerability
dojo/tools/horusec

### CODE REVIEW

File: dojo/tools/gosec/parser.py [hunk 1]
```diff
@@ -17,32 +16,37 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
+            data = json.loads(str(tree, "utf-8"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/gosec/parser.py:20]:
```suggestion
        except Exception:
```

File: dojo/tools/h1/parser.py [hunk 2]
```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
         # Load the contents of the JSON file into a dictionary
         data = file.read()
         try:
-            tree = json.loads(str(data, 'utf-8'))
-        except:
+            tree = json.loads(str(data, "utf-8"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:33]:
```suggestion
        except Exception:
```

File: dojo/tools/h1/parser.py [hunk 3]
```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
...
+                issue_tracker_id = content["attributes"][
+                    "issue_tracker_reference_id"
+                ]
+                issue_tracker_url = content["attributes"][
+                    "issue_tracker_reference_url"
+                ]
+                references = "[{}]({})\n".format(
+                    issue_tracker_id, issue_tracker_url
+                )
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:59]:
```suggestion
            except Exception:
```

File: dojo/tools/h1/parser.py [hunk 4]
```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
...
             # Build the severity of the Dojo finding
             try:
-                severity = content["relationships"]["severity"]["data"]["attributes"]["rating"].capitalize()
+                severity = content["relationships"]["severity"]["data"][
+                    "attributes"
+                ]["rating"].capitalize()
                 if severity not in ["Low", "Medium", "High", "Critical"]:
                     severity = "Info"
-            except:
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:69]:
```suggestion
            except Exception:
```

File: dojo/tools/h1/parser.py [hunk 5]
```diff
@@ -70,11 +82,17 @@ def get_findings(self, file, test):
...
             # Set CWE of the Dojo finding
             try:
-                cwe = int(content["relationships"]["weakness"]["data"]["attributes"]["external_id"][4:])
-            except:
+                cwe = int(
+                    content["relationships"]["weakness"]["data"]["attributes"][
+                        "external_id"
+                    ][4:]
+                )
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:90]:
```suggestion
            except Exception:
```

File: dojo/tools/h1/parser.py [hunk 6]
```diff
@@ -96,43 +114,60 @@ def get_findings(self, file, test):
...
 
         # Try to grab CVSS
         try:
-            cvss = content['relationships']['severity']['data']['attributes']['score']
+            cvss = content["relationships"]["severity"]["data"]["attributes"][
+                "score"
+            ]
             description += "CVSS: {}\n".format(cvss)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:151]:
```suggestion
        except Exception:
```

File: dojo/tools/h1/parser.py [hunk 7]
```diff
@@ -96,43 +114,60 @@ def get_findings(self, file, test):
...
+            weakness_title = content["relationships"]["weakness"]["data"][
+                "attributes"
+            ]["name"]
+            weakness_desc = content["relationships"]["weakness"]["data"][
+                "attributes"
+            ]["description"]
+            description += "\n##Weakness: {}\n{}".format(
+                weakness_title, weakness_desc
+            )
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:170]:
```suggestion
        except Exception:
```

File: dojo/tools/harbor_vulnerability/parser.py [hunk 8]
```diff
@@ -18,11 +18,10 @@ def get_description_for_scan_types(self, scan_type):
         return "Import vulnerabilities from Harbor API."
 
     def get_findings(self, filename, test):
-
         tree = filename.read()
         try:
             data = json.loads(str(tree, "utf-8"))
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/harbor_vulnerability/parser.py:24]:
```suggestion
        except Exception:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/harbor_vulnerability/parser.py (hunk 8 above)
Comment (ajtortolero) [reply dojo/tools/harbor_vulnerability/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py (hunk 7 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py (hunk 5 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py (hunk 4 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py:69]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py (hunk 3 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py (hunk 2 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/gosec/parser.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/tools/gosec/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50

File: dojo/tools/h1/parser.py [hunk 9]
```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
...
             # Build the severity of the Dojo finding
             try:
-                severity = content["relationships"]["severity"]["data"]["attributes"]["rating"].capitalize()
+                severity = content["relationships"]["severity"]["data"][
+                    "attributes"
+                ]["rating"].capitalize()
                 if severity not in ["Low", "Medium", "High", "Critical"]:
                     severity = "Info"
-            except:
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/h1/parser.py:69]:
```suggestion
            except Exception:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/h1/parser.py (hunk 9 above)
Comment (ajtortolero) [reply dojo/tools/h1/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/3bb796aa4ce660190d7acd70d356d89ffee3d772

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by Maffooch** [comment]
Unit tests are being very picky on this one...

**Review by cneill — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**